lang/MoarVM/Portfile

```diff
@@ -22,7 +23,13 @@ long_description    MoarVM is a modern virtual machine built for the Rakudo \
 homepage            https://moarvm.org/
 master_sites        https://moarvm.org/releases/
 
-conflicts           moar
+conflicts moar
```

Comment:
Could we please return the alignment back?

Reply:
Not sure how that got removed, purely unintentional.